UnityProject/Assets/Scripts/Items/PlaceableTile.cs

```diff
@@ -61,6 +61,21 @@ public void ClientPredictInteraction(PositionalHandApply interaction)
 			Cooldowns.TryStartClient(interaction, CommonCooldowns.Instance.Melee);
 		}
 
+		public void Clear()
+		{
+			int i = 0;
+			foreach (var entry in waysToPlace)
+			{
+				if (entry.layerTile is ConnectedTile)
+				{
+					(entry.layerTile as ConnectedTile).Clear();
+					i++;
+				}
+			}
+
+			Debug.Log("Called clear" + i + " times on a connected tile");
```

Comment:
Might be worth cleaning up the log?

Reply:
actually just  use logger.log  instead  and make a custom category called cleaning

Reply:
Done